tasks/check_manifest.rake

```diff
@@ -36,7 +36,7 @@ task :check_manifest => [:templates, "configure"] do
     java/org/yarp/Loader.java
     java/org/yarp/Nodes.java
     java/org/yarp/Parser.java
-    lib/yarp.{jar,so,bundle}
+    lib/yarp/yarp.*
```

Comment:
This could ignore a `lib/yarp/yarp.rb` or `lib/yarp/yarp.config` or so, it's probably safer to keep the extensions there

Reply:
Fair, will change it.